### PR TITLE
Rephrase feedback message for c-tomorrow

### PR DESCRIPTION
**Pull request by yanbingtao**

### CODE REVIEW

**Review by Persdre — DISMISSED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/FindByTagTomorrowCommand.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
     public static final String COMMAND_WORD = "c-tomorrow";
 
     public static final String MESSAGE_SUCCESS = "Tomorrow is %2$s."
-            + "\nThere are total %1$s employees working tomorrow."
+            + "\nThere are total %1$s employees available tomorrow."
```
Comment (ureshiiYing) [inline src/main/java/seedu/address/logic/commands/FindByTagTomorrowCommand.java]: Should be phrased as "There are a total of ... "

Comment (yanbingtao) [reply src/main/java/seedu/address/logic/commands/FindByTagTomorrowCommand.java]: Thanks for the suggestion. will edit it in the latest commit.

**Review by Persdre — APPROVED** [review]
LGTM

**Review by Persdre — APPROVED** [review]
LGTM

**Review by ureshiiYing — APPROVED** [review]
LGTM

**Pull request merged**